pinot-tools/src/main/java/org/apache/pinot/tools/AuthQuickstart.java

```diff
@@ -19,12 +19,17 @@
 package org.apache.pinot.tools;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import org.apache.pinot.core.auth.BasicAuthUtils;
 import org.apache.pinot.spi.plugin.PluginManager;
 
 
 public class AuthQuickstart extends Quickstart {
+  @Override
+  public List<String> types() {
+    return List.of("AUTH");
```

Comment:
We need to be JDK8 source compatible - `Collections.singletonList("AUTH")` would do the trick.

Reply:
Ah I didn't realise that! I did start with `Arrays.asList` and `Collections.singletonList` and IntelliJ was like 'no!' :smile:

Reply:
Ok, have updated that everywhere